### ISSUE TITLE
Add the possibility to find items in containers in the world

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#45]
```
We should add the possibility of finding items in containers in the world. This 
could increase the importance of actually travelling to some maps and places.

I'm thinking we extend the map triggers to also include places where the player 
gets the result of a droplist. This way we can make the items random or 
predefined, and reuse the droplists (for "junk" containers etc.)
```

Original issue reported on code.google.com by `oskar.wi...@gmail.com` on 1 Dec 2010 at 9:34

**Comment by GoogleCodeExporter** [comment]
```
Should this containers refill or are they empty after the first time?
```

Original comment by `SamuelPl...@gmail.com` on 12 Dec 2010 at 2:55

**Comment by GoogleCodeExporter** [comment]
```
That depends on the item. A chest with an emerald would not refill but a 
beehive with honey would. Unless you kill all bees, that is.
```

Original comment by `johan.si...@gmail.com` on 15 Dec 2010 at 11:18

**Comment by GoogleCodeExporter** [comment]
```
Containers are now implemented in the code. They use the same logic as usual 
loot bags, but are just not shown as lootbags on the mainview. Mapobject type 
"container" can now be created, that will use a droplist to determine what 
items should be in the container initially.

Since we are re-using the loot bag logic, they will be persisted in the 
savegame file. Anything you place in the container will be persisted, and any 
contents (partial or full) will also be persisted.

Containers will be stocked with new items the first time you enter the map (or 
load a savegame older than 0.6.8).

When creating maps, we should place graphic objects (chests, item piles) to 
make it obvious that there is something that can be picked up on such tiles.

Now we just need to update the maps to place container mapobjects on the maps 
that should contain them, and crete the corresponding droplists of items.
```

Original comment by `oskar.wi...@gmail.com` on 9 Jan 2011 at 11:01
- Changed state: **Started**
- Added labels: **Milestone-0.6.8**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.8 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 24 Feb 2011 at 9:32
- Changed state: **Fixed**
